frontend/src/container/NewWidget/index.tsx

```diff
@@ -427,52 +421,44 @@ function NewWidget({ selectedGraph }: NewWidgetProps): JSX.Element {
 
 	return (
 		<Container>
-			<div className="facing-issue-btn-container">
-				<FacingIssueBtn
```

Comment:
I will work on adding this back in a separate PR or will push the changes in this one only, whichever seems right that time.